modules/plugins/terminal/toggleterm/toggleterm.nix

```diff
@@ -83,5 +114,19 @@ in {
         open = mkMappingOption "Open lazygit [toggleterm]" "<leader>gg";
       };
     };
+
+    slides = {
+      enable = mkEnableOption "Slides integration";
+
+      package = mkOption {
+        type = nullOr package;
+        default = pkgs.slides;
+        description = "The package that should be used for slides. Setting it to null will attempt to use slides from your PATH";
+      };
```

Comment:
You can use [`mkPackageOption`](https://noogle.dev/f/lib/mkPackageOption) here. Please also spread the desription into multiple lines using multi-line strings.